test/cli/install/__snapshots__/bun-workspaces.test.ts.snap

```diff
@@ -1,1337 +1,2123 @@
+      "resolution": {
+        "tag": "root",
+        "value": "",
+      },
+      "scripts": {},
     },
     {
+      "bin": null,
+      "dependencies": [],
       "id": 1,
-      "name": "lodash",
-      "name_hash": "15298228331728003776",
-      "resolution": "workspace workspace:packages/mono",
-      "dependencies": {},
       "integrity": null,
       "man_dir": "",
+      "name": "lodash",
+      "name_hash": "15298228331728003776",
       "origin": "npm",
-      "bin": null,
-      "scripts": {}
+      "resolution": {
+        "tag": "workspace",
+        "value": "workspace:packages/mono",
+      },
+      "scripts": {},
     },
     {
+      "bin": null,
+      "dependencies": [
+        2,
+      ],
       "id": 2,
+      "integrity": null,
+      "man_dir": "",
       "name": "bar",
       "name_hash": "11592711315645265694",
-      "resolution": "workspace workspace:packages/bar",
+      "origin": "npm",
+      "resolution": {
+        "tag": "workspace",
+        "value": "workspace:packages/bar",
+      },
+      "scripts": {},
+    },
+  ],
+  "trees": [
+    {
       "dependencies": {
+        "bar": {
+          "id": 0,
+          "package_id": 2,
+        },
         "lodash": {
-          "literal": "kjwoehcojrgjoj",
-          "dist_tag": {
-            "name": "lodash",
-            "tag": "lodash"
-          },
-          "resolved_id": 1,
-          "behavior": "normal | workspace"
-        }
+          "id": 1,
+          "package_id": 1,
+        },
       },
-      "integrity": null,
-      "man_dir": "",
-      "origin": "npm",
-      "bin": null,
-      "scripts": {}
-    }
+      "depth": 0,
+      "id": 0,
+      "path": "node_modules",
+    },
   ],
   "workspace_paths": {
     "11592711315645265694": "packages/bar",
-    "15298228331728003776": "packages/mono"
+    "15298228331728003776": "packages/mono",
   },
   "workspace_versions": {
-    "11592711315645265694": "1.0.0"
-  }
-}"
+    "11592711315645265694": "1.0.0",
+  },
+}
 `;
 
 exports[`dependency on workspace without version in package.json: version: *.1.* 1`] = `
-"{
+{
+  "dependencies": [
+    {
+      "behavior": {
+        "workspace": true,
+      },
+      "id": 0,
+      "literal": "packages/bar",
+      "name": "bar",
+      "package_id": 2,
+      "workspace": "packages/bar",
+    },
+    {
+      "behavior": {
+        "workspace": true,
+      },
+      "id": 1,
+      "literal": "packages/mono",
+      "name": "lodash",
+      "package_id": 1,
+      "workspace": "packages/mono",
+    },
+    {
+      "behavior": {
+        "normal": true,
+        "workspace": true,
+      },
+      "id": 2,
+      "literal": "*.1.*",
+      "name": "lodash",
+      "npm": {
+        "name": "lodash",
+        "version": ">=0.0.0",
+      },
+      "package_id": 1,
+    },
+  ],
   "format": "v2",
   "meta_hash": "1e2d5fa6591f007aa6674495d1022868fc3b60325c4a1555315ca0e16ef31c4e",
   "package_index": {
-    "lodash": 1,
+    "bar": 2,
```

Comment:
should this order have flipped everywhere?

Reply:
Taking a snapshot of the object vs string. I think I will switch back to returning a string from zig, then parse it as json in js

Reply:
i kinda like it tbh

but also i think it would catch determinism issues potentially